foundation/eclipselink.core.test/src/org/eclipse/persistence/testing/tests/TestRunModel.java

```diff
@@ -13,13 +13,17 @@
 package org.eclipse.persistence.testing.tests;
 
 import java.lang.reflect.Method;
-import java.util.*;
-
-import junit.framework.Test;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Vector;
 
 import org.eclipse.persistence.internal.helper.Helper;
 import org.eclipse.persistence.sessions.DatabaseLogin;
-import org.eclipse.persistence.testing.framework.*;
+import org.eclipse.persistence.testing.framework.TestModel;
+
+import junit.framework.Test;
 
```

Comment:
Imports only change? Why?

Reply:
`git cherry-pick` found out that line having `AdvancedProcessingTestModel` has been already removed from the file in 2.7 by some earlier commit

Reply:
Thanks for the response. Changes are already approved so go on with merging it.